src/2d/dig/Makefile.dclaw

```diff
@@ -23,14 +23,14 @@ COMMON_MODULES += \
  $(GEOLIB)/multilayer/multilayer_module.f90 \
  $(GEOLIB)/friction_module.f90 \
  $(DIGLIB)/refinement_module.f90 \
- $(GEOLIB)/adjointsup_module.f90
+ $(GEOLIB)/adjointsup_module.f90 \
+ $(GEOLIB)/fgout_module.f90 \
 
 #list of common modules needed for dclaw codes
 COMMON_MODULES += \
  $(DIGLIB)/digclaw_module.f90 \
  $(DIGLIB)/qinit_module.f90 \
  $(DIGLIB)/auxinit_module.f90 \
- $(DIGLIB)/fgout_module.f90 \
```

Comment:
@rjleveque , @dlgeorge I suggest we delete dclaw/src/2d/dig/fgout_module.f90.

We only have it it as a placeholder that is no longer needed b/c Randy updated the GEOLIB/fgout_module.f90. 

Thoughts?

Reply:
seems best to remove it I guess.

Reply:
I think that's fine.  The only difference in it was that it set
```
fg%num_vars(1) = 10 ! DIG different number
```
but this is no longer needed with the geoclaw updates.